### PR TITLE
Proposal for Limiting files under 10M, minor formatting change.

### PR DESCRIPTION
**Pull request by wagerpascal**
I found a possible solution that is also in the OS library at http://stackoverflow.com/questions/8626325/most-efficient-way-to-delete-a-file-if-its-below-a-certain-size that may be able to limit the file size to only above 10M.

Haven't been able to test it (see #1 ).

### CODE REVIEW

**Comment by mysteriouss** [comment]
HTTP has its HEAD method to receive the content size of future responses, e.g.

curl -X HEAD 'http://yinyueshiting.baidu.com/data2/music/122898508/10599428720001045.flac?xcode=951d436d8eaacc99e4d6351b3c1b445d'
curl: (18) transfer closed with 35660809 bytes remaining to read

however may not be strictly followed.

File: main.py [hunk 1]
```diff
@@ -55,6 +55,8 @@
     songname = d["data"]["songList"][0]["songName"]
     artistName = d["data"]["songList"][0]["artistName"]
     filename = "./" + songdir + "/"+songname+"-"+artistName+".flac"
+    if os.path.getsize(filename) < 10000 * 1024: #Source from http://stackoverflow.com/questions/8626325/most-efficient-way-to-delete-a-file-if-its-below-a-certain-size
```
Comment (YongHaoWu) [inline main.py]: The position of this code is not right.
We should check the file size and delete it after downloading.

Comment (wagerpascal) [inline main.py]: Moved it to the bottom.

**Comment by YongHaoWu** [comment]
@wagerpascal Hi, your commit is conflict now. Could you please make  a new patch according the new main.py? If you are busy, I can do that for you.

**Comment by YongHaoWu** [comment]
@mysteriouss You are right. See this commit: https://github.com/YongHaoWu/NeteaseCloudMusicFlac/commit/2f20433cd4643a1ae466b4130a79cd80a59b8f18#diff-5bc02cefb3ea9e27f1a6776eabd1935dR65

+        headers = requests.head(songlink).headers
 +        size = int(headers['Content-Length']) / (1024 ** 2)
Thank you :)

**Pull request closed without merging**